omero/developers/Model.txt

```diff
@@ -385,19 +382,19 @@ Each instance can be found in one of several states. Quickly, they are:
     :alt: Object states
 
 Identity, references, and versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Critical for understanding these states is understanding the concepts of
 identity and versioning as it relates to ORM. Every object has an id
-field that if created by the backend will not be :literal:`null`. However, every
-table does not have a primary key field -- subclasses contain a foreign
+field that if created by the backend will not be :literal:`null`. However,
```

Comment:
is it not intended to be "that, if created by the backend, will not be…"? Changing it to 'is' stops the 'will not be' bit making sense.